File: api/liste/liste.java
package com.oguzhan.api.liste;

import jakarta.validation.constraints.NotEmpty;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.autoconfigure.AutoConfiguration;
import org.springframework.data.annotation.Id;

import java.sql.Time;
import java.sql.Timestamp;
import java.time.LocalDateTime;

public record liste(
        @Id
        Integer id,
        String isim,
        Integer film_sayisi,
        String aciklama,
        Float ortalama_puan,
        Integer sahipid,
        String olusturulma_tarihi

) {}

File: veritabaniodevapp/model/liste.kt
package com.example.veritabaniodevapp.model


data class liste(
    var isim : String ,
    var film_sayisi : Int,
    var aciklama : String ,
    var ortalama_puan : Float ,
    val sahipid : Integer ,
    val olusturulma_tarihi : String,
    val id : Int
)